devops/Makefile

```diff
@@ -456,6 +459,9 @@ deploy-queue-consumers:
 		-p ATBC_ENDPOINT=$(ATBC_ENDPOINT) \
 		-p APPLICATION_ARCHIVE_DAYS=$(APPLICATION_ARCHIVE_DAYS) \
 		-p IS_FULLTIME_ALLOWED=$(IS_FULLTIME_ALLOWED) \
+    -p CAS_BASE_URL=$(CAS_BASE_URL) \
+    -p CAS_CLIENT_ID=$(CAS_CLIENT_ID) \
+    -p CAS_CLIENT_SECRET=$(CAS_CLIENT_SECRET) \
```

Comment:
I believe the secrets are not properly mapped. Please take a look at a similar secret mapped to queue-consumers and follow the same approach.
https://github.com/bcgov/SIMS/blob/f3d5fa0250717a3693aec48f7883439e23b68bd1/devops/openshift/queue-consumers-deploy.yml#L137-L141